src/main/java/seedu/address/model/cheese/CheeseType.java

```diff
@@ -23,7 +23,7 @@
      *
      * @param type A valid cheese type.
      */
-    private CheeseType(String type) {
+    public CheeseType(String type) {
```

Comment:
Would be good to leave it as private and use the static method

Reply:
done!